packages/aurora-save/index.js

```diff
@@ -1,24 +1,10 @@
-import Jetpack from "fs-jetpack";
-const os = require("os"); // For some reason I cannot import using normal syntax so I have to do it this way.
-import { convertToRaw, convertFromRaw, EditorState } from "draft-js";
-
-const noteFileExt = ".aur";
-
-function getAuroraDirContext() {
-  return Jetpack.dir(os.homedir()).dir(".aurora");
-}
-
-function attachMetaData(editorState) {
-  const note = {
-    contentState: convertToRaw(editorState.getCurrentContent()),
-    date: new Date().getTime()
-  };
-  return note;
-}
-
-function noteFileName(note) {
-  return note.date + noteFileExt; // name based on date to ensure uniqueness
-}
+import {
+  getAuroraDirContext,
+  attachMetaData,
+  noteFileName,
+  noteFileExt
+} from "./util.js";
+import { convertFromRaw, EditorState } from "draft-js";
```

Comment:
This is probably a good idea to break up aurora-save and its utils.

Reply:
Cool <3 Though it was mostly just for testing. Because I didn't want to export those utility functions outside the module.